python/lsst/faro/measurement/DetectorTableMeasurement.py

```diff
@@ -82,12 +77,17 @@ class DetectorTableMeasurementTask(CatalogMeasurementBaseTask):
 
     def runQuantum(self, butlerQC, inputRefs, outputRefs):
         inputs = butlerQC.get(inputRefs)
-        catalog = inputs["catalog"].get(parameters={"columns": self.config.columns})
+        kwargs = {}
+        kwargs["currentBands"] = None
```

Comment:
Might be useful to add a docstring/comment to explain why this strange construction of setting currentBands=None is necessary.

Reply:
Added 

> """currentBands is set to None in sourceTable contexts, because currentBands is used to 
        provide the correct parquet column names """